lib/gui/modules/drive-scanner.js

```diff
@@ -110,9 +95,16 @@ const scan = () => {
       includeSystemDrives: settings.get('unsafeMode')
     },
     usbboot: {
-      files: {
-        'bootcode.bin': USBBOOT_BOOTCODE_BIN_BUFFER,
-        'start.elf': USBBOOT_START_ELF_BUFFER
+      readFile: (name) => {
+        const blobPath = _.attempt(() => {
+          if (name === 'bootcode.bin' || name === 'start_cd.elf') {
```

Comment:
fixup_cd.dat ought to be in the same folder as start_cd.elf

Reply:
Done!